services/ossf-scorecard/ossf-scorecard.service.js

```diff
@@ -0,0 +1,59 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+import { colorScale } from '../color-formatters.js'
+
+const schema = Joi.object({
+  score: Joi.required(),
```

Comment:
This is a numeric, >=0 value field in the raw response correct? If so let's reflect that in that in the schema